src/sentry/integrations/vsts/issues.py

```diff
@@ -32,13 +32,13 @@ def get_persisted_default_config_fields(self) -> Sequence[str]:
 
     def create_default_repo_choice(self, default_repo: str) -> tuple[str, str]:
         # default_repo should be the project_id
-        project = self.get_client(base_url=self.instance).get_project(default_repo)
```

Comment:
all of the `get_client` calls in here pass `self.instance `which is the default in `get_client`